src/components/main/Task.tsx

```diff
@@ -95,78 +101,86 @@ const Task = (props: TaskProps): JSX.Element => {
     membersList.find((memberInList) => m === memberInList.id)?.name ||
     t('Unknown');
 
-  const { attributes, listeners, setNodeRef, transform, isDragging } =
-    useDraggable({
-      id,
-      data: {
-        type,
-      },
-      disabled: dialogOpen,
-    });
-
-  const transformation: React.CSSProperties = {
-    transform: CSS.Translate.toString(transform),
-  };
+  const renderTask = (isDragging: boolean): JSX.Element => (
+    <TaskCard
+      onDragOver={(e) => onDragOver(e)}
+      onDrop={(e) => onDrop(e)}
+      raised={isDragging}
+      sx={{
+        position: 'relative',
+        zIndex: isDragging ? 2 : 'auto',
+      }}
+      key={key}
+      onClick={(event) => {
+        event.stopPropagation();
+      }}
+    >
+      <CardHeader
+        avatar={
+          <AvatarGroup max={3}>
+            {members.map((member) => (
+              <Avatar
+                key={member}
+                sx={{
+                  bgcolor: `${getMemberColor(member)}`,
```

Comment:
You seem to use the `members.find()` multiple times for the same member. 
Solutions: 
- find your member from memberList once at the start of your `map`
- change `memberList` to be an object with the `memberId` as keys so you can access the member in constant time with `memberObject[member.id]`. When you will need to iterate over your memberObject you can always use the `Object.entries(memberObject)` method to get an array that you can map over.